Index: app.ui/src/test/java/tdt4140/gr1817/app/ui/FxAppTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package tdt4140.gr1817.app.ui;\n\nimport org.junit.Test;\nimport org.testfx.framework.junit.ApplicationTest;\n\nimport javafx.fxml.FXMLLoader;\nimport javafx.scene.Parent;\nimport javafx.scene.Scene;\nimport javafx.stage.Stage;\n\npublic class FxAppTest extends ApplicationTest {\n\n\t@Override\n    public void start(Stage stage) throws Exception {\n        Parent root = FXMLLoader.load(getClass().getResource(\"FxApp.fxml\"));\n        Scene scene = new Scene(root);\n        stage.setScene(scene);\n        stage.show();\n    }\n\n    @Test\n    public void testFxApp() {\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app.ui/src/test/java/tdt4140/gr1817/app/ui/FxAppTest.java	(revision 02f4a5dc89e7498378d589e5173f4a21ea76e5be)
+++ app.ui/src/test/java/tdt4140/gr1817/app/ui/FxAppTest.java	(date 1516744713526)
@@ -1,5 +1,6 @@
 package tdt4140.gr1817.app.ui;
 
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.testfx.framework.junit.ApplicationTest;
 
@@ -10,7 +11,20 @@
 
 public class FxAppTest extends ApplicationTest {
 
-	@Override
+    @BeforeClass
+    public static void headless() {
+        System.setProperty("prism.verbose", "true");
+        System.setProperty("java.awt.headless", "true");
+        System.setProperty("testfx.robot", "glass");
+        System.setProperty("testfx.headless", "true");
+        System.setProperty("glass.platform", "Monocle");
+        System.setProperty("monocle.platform", "Headless");
+        System.setProperty("prism.order", "sw");
+        System.setProperty("prism.text", "t2k");
+        System.setProperty("testfx.setup.timeout", "2500");
+    }
+
+    @Override
     public void start(Stage stage) throws Exception {
         Parent root = FXMLLoader.load(getClass().getResource("FxApp.fxml"));
         Scene scene = new Scene(root);
